inst/rmd/h/default.html

```diff
@@ -51,15 +51,23 @@
 <style type="text/css">code{white-space: pre;}</style>
 <style type="text/css">
 $highlighting-css$
+
 $if(code_folding)$
-div.sourceCode {
-  overflow-x: visible;
+
+.row {
+  width: 100%;
```

Comment:
It felt a bit unsafe doing this. While this specific change opened the door to a fix, it's a bit naive since I don't know all of the implications. Perhaps I can explore the use of a plain `<div>` with `text-align: right` as you've suggested above.